pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -170,7 +170,7 @@ func RunListBuildWebHooks(f *clientcmd.Factory, out, errOut io.Writer, name stri
 		}
 		ref := build.Config
 		if ref == nil {
-			return fmt.Errorf("the provided build %q was not created from a build config and cannot have webhooks", name)
+			return fmt.Errorf("the provided build %q was not created from a buildConfig and cannot have webhooks", name)
```

Comment:
build configuration - for end user messages we should never use "BuildConfig" or "DeploymentConfig", but instead "build configuration" or "deployment configuration".  "build config" or "deployment config" is acceptable in a paragraph where you've already used "build configuration".